roles/gitian/templates/gitian-build.sh

```diff
@@ -215,7 +215,11 @@ then
             echo ""
             echo "Compiling variant: ${VERSION}_${suite}"
             echo ""
-
+            #workaround python and python3 in buster
+            if [[ $suite = "buster" ]]
+            then
+                sed -i -e 's/- "python3"/- "python"/g' -e 's/- "python-is-python3"//g' ${suite_dir_path}/gitian-linux-parallel.yml;
```

Comment:
Would it be more maintainable to fix this in `gitian-linux-parallel.yml` itself?